src/test/groovy/org/jenkinsci/gradle/plugins/jpi/JpiPomCustomizerSpec.groovy

```diff
@@ -266,7 +267,11 @@ class JpiPomCustomizerSpec extends Specification {
         VersionSelectorScheme versionSelectorScheme = new DefaultVersionSelectorScheme(versionComparator)
         VersionRangeMapper versionRangeMapper = new MavenVersionRangeMapper(versionSelectorScheme)
         MavenPomFileGenerator pomGenerator = new MavenPomFileGenerator(pomInternal.projectIdentity, versionRangeMapper)
-        pomGenerator.packaging = pomInternal.packaging
+        if (GradleVersion.version(project.gradle.gradleVersion) > GradleVersion.version('4.7')) {
+            pomGenerator.model.packaging = pomInternal.packaging
```

Comment:
Instead of using the MavenPomFileGenerator, we could probably also create the pom file by hand. WDYT?